editor/modes/visual-editor/block-list.js

```diff
@@ -46,6 +57,28 @@ class VisualEditorBlockList extends wp.element.Component {
 	componentWillUnmount() {
 		document.removeEventListener( 'copy', this.onCopy );
 		document.removeEventListener( 'cut', this.onCut );
+
+		// Cancel throttled calls.
+		this.onPointerMove.cancel();
+	}
+
+	setBlockRef( ref, uid ) {
+		if ( ref === null ) {
+			delete this.refs[ uid ];
+		} else {
+			this.refs = {
+				...this.refs,
+				[ uid ]: ref,
+			};
+		}
+	}
+
+	onPointerMove( { clientY } ) {
+		const y = clientY + window.pageYOffset;
+		const key = this.coordMapKeys.reduce( ( acc, topY ) => y > topY ? topY : acc );
```

Comment:
For my own curiosity, I wanted to see whether there was really any benefit to assigning a separate array property than just iterating the object keys here. Was surprised to find that the native `Array#reduce` is a fair bit faster than Lodash's `reduce` on the object.

https://jsperf.com/lodash-object-keys-vs-array-keys/1

In other words, some merit to keeping this as-is 👍